content/docs/guides/scim/okta.md

```diff
@@ -0,0 +1,136 @@
+
+Once the feature is enabled, navigate to the Pulumi SAML app configured in your Okta org from the **Classic UI**. Under the **General** tab, in the **App Settings** dialog box, locate the **Provisioning** section and select **SCIM**. Click **Save** to save your changes and close the **App Settings** dialog box.
+
+## Changing the SAML SSO Settings For SCIM
+
+As part of the setup process, you need to adjust some of the parameters sent by a SAML SSO request. To make the changes, navigate to the **SAML Settings** section, click **Edit**, and make the following changes:
+
+1. Click **Next** on the first step.
+2. On Step 2, in the **General** section, set the application username to **Okta username prefix**.
+3. In the **Attribute Statements** section, click **Add Another** and set the **Name** of the property to `email` and the value to be source from `user.email`. The ATTRIBUTE STATEMENTS should now be as follows:
+
+    | Name | Name Format | Value |
+    | --- | --- | --- |
+    | firstName | Basic | user.firstName |
+    | lastName | Basic | user.lastName |
+    | email | Basic | user.email |
+
+4. Click **Next** then click **Finish**.
+
+## Configuring the SCIM Connector
+
+To configure the SCIM connector, click the **Provisioning** tab, and then select the **Integration** option in the left nav. Click **Edit**. Fill out the form as follows:
+
+* SCIM connector base URL: `https://api.pulumi.com/scim/v2/{orgName}`, where `{orgName}` must be replaced with your organization’s login name (not display name). If you do not know this, navigate to your SAML settings and look at the SSO URL. It will have your organization’s login name in the URL.
+* Supported provisioning actions: check **Push New Users** and **Push Profile Updates**.
+* Authentication Mode: HTTP Header
+* For **HTTP HEADER**, you will used a token from the [Pulumi Console](https://app.pulumi.com) as the authorization bearer token. To generate a token, navigate to your org in the Pulumi Console, click on the **Settings** tab, and then click **SAML SSO**. Scroll down to the **SCIM** section and generate a new token if you have never generated one for your org, or regenerate it if you have already done so in the past.
+
+    {{% notes "info" %}}
+Once you generate the token, please save it securely. The Pulumi Console will not show you the token again and there is no way to retrieve it again. SCIM token can only be regenerated once it has been generated already. This means you will invalidate any previous SCIM tokens for your org by regenerating it.
+    {{% /notes %}}
+
+* Paste the token from the Pulumi Console into the Okta SCIM configuration for **HTTP HEADER**.
+
+The following shows how your SCIM connection info should be filled in.
+
+![Okta SCIM provisioning](/images/docs/guides/scim/okta-scim-provisioning.png)
+
+## Enabling SCIM Provisioning Actions
+
+There are a few provisioning actions you need to enable, which Okta will con on the Pulumi service.
```

Comment:
There seems to be something missing in the second part of this sentence.

Reply:
```suggestion
To configure the actions that Okta will send to the Pulumi Console, you need to enable them.
```